blamo/lib/Test/test.dart

```diff
@@ -51,29 +51,96 @@ class _TestPageState extends State<TestPage> {
                   FormBuilderTextField(
                     attribute: 'type-no',
                     validators: [],
-                    decoration: InputDecoration(labelText: "Type Number"),
+                    decoration: InputDecoration(labelText: "Test Type And Number"),
```

Comment:
The new values on test page don't line up with the values accepted by the PDF. The PDF is created with fields from Victoria's expected data.

test.dart:
Test Type and Number
Depth Upper Bound
Depth Lower Bound
Percent Recovery
Soil Driving Resistance / Rock Discontinuity
Percent Natural Moisture
Material Description


PDF test class
beginTestDepth
endTestDepth
soilType
description
moistureContent
dryDensity
liquidLimit
plasticLimit
fines
blows1
blows2
blows3
blowCount